docs/topics/request-response.rst

```diff
@@ -115,20 +115,8 @@ Request objects
         cookies for that domain and will be sent again in future requests.
         That's the typical behaviour of any regular web browser.
 
-        To create a request that does not send stored cookies and does not
-        store received cookies, set the ``dont_merge_cookies`` key to ``True``
-        in :attr:`request.meta <scrapy.Request.meta>`.
-
-        Example of a request that sends manually-defined cookies and ignores
-        cookie storage:
-
-        .. code-block:: python
-
-            Request(
-                url="http://www.example.com",
-                cookies={"currency": "USD", "country": "UY"},
-                meta={"dont_merge_cookies": True},
-            )
+        It is recommended not to set the ``dont_merge_cookies`` key to ``True``
+        in :attr:`request.meta <scrapy.Request.meta>` as it causes custom cookies to be ignored.
```

Comment:
I would change the wording, as I think it could be understood as a general recommendation rather than something that only applies when custom cookies are set (even though it can be inferred from context), and it would be good to link `dont_merge_cookies` to its docs (even though the previous text did not).

What about this?

```suggestion
        Note that setting the :reqmeta:`dont_merge_cookies` key to ``True`` in
        :attr:`request.meta <scrapy.Request.meta>` causes custom cookies to be
        ignored.
```

Reply:
Yeah I'll change it and also add the link

Reply:
@Gallaecio is this the right [link](https://doc.scrapy.org/en/latest/topics/downloader-middleware.html#cookies-enabled) for the `dont_merge_cookies` docs

Reply:
`:reqmeta:`dont_merge_cookies` will generate a link, it’s a special syntax we have in Scrapy docs for request metadata keys.